osbenchmark/worker_coordinator/runner.py

```diff
@@ -1102,6 +1103,14 @@ async def __call__(self, opensearch, params):
     def __repr__(self, *args, **kwargs):
         return "put-pipeline"
 
+# TODO: refactor it after python client support search pipeline https://github.com/opensearch-project/opensearch-py/issues/474
+class CreateSearchPipeline(Runner):
```

Comment:
Since the codebase uses `put-pipeline`, `class PutPipeline`, `workload.OperationType.PutPipeline`, and `OperationType.PutPipeline` for the [ingest pipeline API](https://opensearch.org/docs/2.0/api-reference/ingest-apis/create-update-ingest/), `search-pipeline` should follow the same naming convention.

Could you consider updating the following:
- `create-search-pipeline` --> `search-pipeline`
- `class CreateSearchPipeline(Runner)` --> `class SearchPipeline(Runner)`
- `workload.OperationType.CreateSearchPipeline` --> `workload.OperationType.SearchPipeline`
- `OperationType.CreateSearchPipeline` --> `OperationType.SearchPipeline`

Reply:
Other operations, such as update pipeline (POST) and delete pipeline (DELETE), are not yet supported. I'm thinking if we should rename the `put-pipeline` of ingest pipeline to differentiate it better.

Reply:
Sure, renaming put-pipeline works as well.